### DISCUSSION TITLE
Tailwind v4 support?

### DISCUSSION THREAD

**Discussion by provokcx** [barvian/fluid-tailwind#51] [Help]
Is fluid-tailwind ready for v4?

**Comment by barvian** [comment]
Hi, definitely not 😅. It's using some internal Tailwind stuff and a deprecated v2 API so I'm expecting it will need a pretty big refactor. On a positive note, v4's reliance on `@property` might let me work around the current CSS limitations 👋

**Reply by provokcx** [reply]
Okay, I understand 👌 
Do you plan to work on this as soon as v4 is out with some documentation?

**Reply by barvian** [reply]
Yes for sure! I personally use `fluid-tailwind` all the time 👍

**Reply by provokcx** [reply]
Yes, same. That's why I'm asking. 
I don't want to go back to the way it was before 😂 
Thx a lot for the good work 💪

**Comment by joffreypersia** [comment]
Is the support of tailwindcss V4 on the pipes ? Thank you
